specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/virtualWan.json

```diff
@@ -3623,6 +3623,11 @@
           "format": "int32",
           "description": "Routing weight for vpn connection."
         },
+        "dpdTimeoutSeconds": {
+          "type": "integer",
+          "format": "int32",
+          "description": "DPD timeout for a vpn connection in seconds."
```

Comment:
>DPD [](start = 26, length = 3)

Spelling this out in the description would be more helpful to users

Reply:
Done. DPD = Dead Peer Detection